processor/resourcedetectionprocessor/internal/gcp/documentation.md

```diff
@@ -19,6 +19,9 @@
 | faas.version | The faas.version | Any Str | true |
 | gcp.cloud_run.job.execution | The Job execution name | Any Str | true |
 | gcp.cloud_run.job.task_index | The Job execution task index | Any Str | true |
+| gcp.gce.instance.group.manager.name | The name of an instanceGroupManager. | Any Str | true |
```

Comment:
@jsuereth and I went back and forth on this, but we ended up with dots because it's the "manager" of a "group" of "instance"s. In that scheme an instance group's name would be `gcp.gce.instance.group.name`.

Reply:
discussed offline.  This makes sense, and is ergonomic.